packages/commonwealth/client/scripts/views/pages/NotificationSettings/CommentSubscriptions.tsx

```diff
@@ -17,9 +18,18 @@ export const CommentSubscriptions = () => {
     [threadsFilter],
   );
 
+  const data = commentSubscriptions.data || [];
+
+  if (!data.length) {
+    return (
+      <CWText type="h4" className="error-content">
```

Comment:
same as above

Reply:
It's not a full error page. It's just an 'empty state' . We should use ErrorPage for just an empty state?